cutter/cutter/main.py

```diff
@@ -494,7 +494,16 @@ def main(dbconnect, youtube_creds_file, name=None, base_dir=".", metrics_port=80
 	# We have two independent jobs to do - to perform cut jobs (cutter),
 	# and to check the status of transcoding videos to see if they're done (transcode checker).
 	# We want to error if either errors, and shut down if either exits.
-	dbmanager = DBManager(dsn=dbconnect)
+	dbmanager = None
+	stopping = gevent.event.Event()
```

Comment:
bad copy-paste, I think. We define `stop` as an event in this file already, you should use that.

Reply:
I don't quite follow. `stop` is defined as a handler function not an event unless I am missing something about gevents (which is quite likely)

Reply:
```
stop = gevent.event.Event()
```
(line 489)

an Event is basically a boolean flag that can be wait()ed on. The `event.set()` function sets it to true and wakes up all waiters. On line 490 we're registering `stop.set()` to be called upon recieving a SIGTERM, and this will cause `stop.wait()` to immediately return.

Reply:
Ah, I see. I was confusing how the cutter and sheetsync do this with how the thrimshim does.